Makefile

```diff
@@ -91,7 +94,7 @@ release-delete:
 .PHONY: release-delete
 
 bundle-mastermind:
-	$(ENV_BIN)/pyinstaller mastermind.spec
+	$(ENV_BIN)pyinstaller mastermind.spec
```

Comment:
`ENV_BIN` contains the `/`.  It simplifies the logic when `ENV_BIN` is not defined.